docs/docs/administration/install-from-scratch.md

```diff
@@ -171,18 +171,22 @@ Then press <kbd>Enter</kbd> to customize this guide with your domain name.
 
         !!! example
             Your `.env` file should look something like this
-            
+
             ```
-            SECRET_KEY=REPLACE_THIS_WITH_YOUR_50_CHAR_RANDOMLY_GENERATED_STRING
-            ALLOWED_HOSTS=xDOMAIN_NAMEx
-            DOMAIN_NAME=xDOMAIN_NAMEx
-            POSTGRES_DB=mathesar_django
-            POSTGRES_USER=mathesar
-            POSTGRES_PASSWORD=REPLACE_THIS_WITH_APPROPRIATE_PASSWORD_FOR_THE_CHOSEN_POSTGRES_USER
-            POSTGRES_HOST=localhost
-            POSTGRES_PORT=5432
+            SECRET_KEY="REPLACE_THIS_WITH_YOUR_50_CHAR_RANDOMLY_GENERATED_STRING"
+            ALLOWED_HOSTS="xDOMAIN_NAMEx"
+            DOMAIN_NAME="xDOMAIN_NAMEx"
+            POSTGRES_DB="mathesar_django"
+            POSTGRES_USER="mathesar"
+            POSTGRES_PASSWORD="REPLACE_THIS_WITH_APPROPRIATE_PASSWORD_FOR_THE_CHOSEN_POSTGRES_USER"
+            POSTGRES_HOST="localhost"
+            POSTGRES_PORT="5432"
```

Comment:
@Anish9901 @mathemancer This will break our install from scratch setup. The way we handle env variables would consider the quotes and not omit them.

For eg., `ALLOWED_HOSTS="*"` would not actually consider `*` but would consider `"*"` which would cause all requests to fail.